net/yggdrasil-jumper/Makefile

```diff
@@ -0,0 +1,45 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yggdrasil-jumper
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/one-d-wide/yggdrasil-jumper/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=d0fd59e9f7a110094c9189b49b812fb0014c6c45535b4d30effd7cf602961454
+
+PKG_MAINTAINER:=one d wide <one-d-wide@protonmail.com>
```

Comment:
I am still not sure that this is real name, right?
1. Same applies here and as well in SoB.

Reply:
I'd like to advocate that "real name" (as of linux contributing guidelines, which was adopted by openwrt) has been intended only to make possible tracking change authors and, despite poor wording, had never been actually used to discriminate pseudonyms ([source], see commit description):

> Long long ago, in a more innocent time, Greg wrote the clarification for
> how the DCO should work and that you couldn't make anonymous
> contributions, because the sign-off needed to be something we could
> check back with.
> 
> It was 2006, and nobody reacted to the wording, the whole Facebook 'real
> name' controversy was a decade in the future, and nobody even thought
> about it.  And despite the language, we've always accepted nicknames and
> that language was never meant to be any kind of exclusionary wording.
> 
> In fact, even when it became a discussion in other adjacent projects,
> apparently nobody even thought to just clarify the language in the
> kernel docs, and instead we had projects like the CNCF that had long
> discussions about it, and wrote their own clarifications [1] of it.

The wording of this rule has been recently clarified to avoid causing unnecessary angst and pain ([source], see changes of the same commit):

```diff 
-using your real name (sorry, no pseudonyms or anonymous contributions.)
+using a known identity (sorry, no anonymous contributions.)
```

Also quote from [CNCF guideline] on Developer Certificate of Origin (DCO, which is the concept underlying SoB):

> ## DCO and Real Names
> 
> The DCO requires the use of a real name that can be used to identify someone in case there is an issue about a contribution they made. 
> 
> **A real name does not require a legal name, nor a birth name, nor any name that appears on an official ID (e.g. a passport). Your real name is the name you convey to people in the community for them to use to identify you as you. The key concern is that your identification is sufficient enough to contact you if an issue were to arise in the future about your contribution.**
> 
> Your real name should not be an anonymous id or false name that misrepresents who you are.

[source]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/Documentation/process/submitting-patches.rst?id=d4563201f33a022fc0353033d9dfeb1606a88330
[CNCF guideline]: https://github.com/cncf/foundation/blob/659fd32c86dc/dco-guidelines.md

Reply:
I think, I will be strict on this one. I dont want to argue with you, everybody is not afraid of his known identity or real name. We follow what is done by OpenWrt core developers. If you want to argue about it, then feel free to raise issue in their mailing list. So, since you try to reinvent a wheel here, then how to get it merged:

1. Use your real name

Anything else won't be accepted until you discussed it elsewhere not here. Rules and policy are set. They exists for a while.

Reply:
Hi. Thank for your advice. I mentioned the issue on mailing list and, thanks to Paul Spooren, [Submission Guidelines] identity rule on openwrt wiki is now in accordance with linux's one.

[Submission Guidelines]: https://openwrt.org/submitting-patches#submission_guidelines

And also appropriate CI patch should hopefully land soon #23084 :sparkles:

Reply:
Can you gents resolve this conversation now that #23084 is merged?